build-system/tasks/helpers.js

```diff
@@ -0,0 +1,746 @@
+/**
```

Comment:
A `helpers.js` file with 746 lines seems like it should be further split into sub-files. wdyt?

Reply:
This is already part of the plan. However, further splitting `helpers.js` requires significant code / logic refactoring, which is out of scope for this PR.